### PR TITLE
Finite iterative ordinals

### PR DESCRIPTION
**Pull request by Seiryn21**
Two files of addition, I'd like to know where they should go : 
Similarly to Multiset-addendum and Set-addendum, File1 define ordinals 0 and 1.
File2 define all the finite iterative ordinals.

Also line 57 in File2 is too long, but I don't know where I should break it.

### CODE REVIEW

**Review by martinescardo — COMMENTED** [review]
Hi @Seiryn21 . 

@tomdjong and I discussed this morning how we are going to do this. We are going to reorganize the files in the directory `Iterative` in a better way, so that in particular it will be easier to find a better home for your contributions. I will report back when this is done.

For the moment, I am going to do some minor suggestions regarding your two draft files.

File: source/Various/File2.lagda
Comment (martinescardo) [inline source/Various/File2.lagda:29]: Two comments here. (1) We should think of a better name for this definition (I thought a bit but haven't come up with a good idea yet).

(2) Please use M,N to range over 𝕄, and A,B,C to range over 𝕍, and α, β, γ  to range over 𝕆, for the sake of consistency with the other files and the published literature.

File: source/Various/File2.lagda
Comment (martinescardo) [inline source/Various/File2.lagda:41]: A better name here is also required. Following the unspoken convention in `TypeTopology`, the canonical name would be ℕ-to-𝕄.

File: source/Various/File2.lagda
Comment (martinescardo) [inline source/Various/File2.lagda:62]: I suggest `ℕ-to-𝕄-gives-iset` as a probably better name.

File: source/Various/File2.lagda
Comment (martinescardo) [inline source/Various/File2.lagda:69]: I suggest `ℕ-to-𝕍`.

File: source/Various/File2.lagda
Comment (martinescardo) [inline source/Various/File2.lagda:72]: `preserves` instead of `preserve`.

File: source/Various/File2.lagda
Comment (martinescardo) [inline source/Various/File2.lagda:87]: preserves.

File: source/Various/File2.lagda
Comment (martinescardo) [inline source/Various/File2.lagda:112]: I suggest `ℕ-to-𝕍-gives-iordinal`.

File: source/Various/File2.lagda
Comment (martinescardo) [inline source/Various/File2.lagda:116]: I suggest `ℕ-to-𝕆`.

**Comment by martinescardo** [comment]
Thanks again. It's very nice to have that. I like the way you've structured it.

Once we've done the reorganization mentioned above, I'll get back to you. I am not sure when I am going to be able to do it, because of teaching and other commitments, but I want to do it asap.

File: source/Various/File2.lagda
Comment (martinescardo) [inline source/Various/File2.lagda:57]: I would break it like this
```
III = disjoint-cases-embedding _ _ 
        (𝕄-forest-is-embedding m is-iset) 
        II 
        (λ x ⋆ → φ≠ssupφ m x)
```
or like this
```
III = disjoint-cases-embedding _ _ 
        (𝕄-forest-is-embedding m is-iset) II (λ x ⋆ → φ≠ssupφ m x)        
```
I think I prefer the first one, but both are acceptable.

File: source/Various/File2.lagda
Comment (Seiryn21) [reply source/Various/File2.lagda]: I replaced `φ≠ssupφ` by `𝕄-dont-contain-themselves : (M : 𝕄) → ¬ (M ⁅ M)` which imply it and is more intuitive, I think.

**Comment by ayberkt** [comment]
@Seiryn21 Any plans to finish this?

**Comment by Seiryn21** [comment]
@ayberkt I would be really happy to finish this, but I was told by @martinescardo to wait until he and @tomdjong figured where the content of those file should live. I suppose they never got the time.

**Comment by ayberkt** [comment]
@Seiryn21 I see, thanks for the update. I guess either @martinescardo or @tomdjong will respond soon.

**Comment by ayberkt** [comment]
@Seiryn21 In the meanwhile, could you please address the remaining points that @martinescardo brought up in his review so that we can be ready to merge this once the file it should live in has been decided.

Thanks!

**Comment by Seiryn21** [comment]
@ayberkt I think I addressed them all ? I didn't marked them as resolved because I wasn't sure I should be the one doing it. (If somehow I missed one, sorry)

**Comment by martinescardo** [comment]
Oh, sorry. It is more a memory problem than a time problem. I'll try to look at this tomorrow!

File: source/Various/File2.lagda
Comment (tomdjong) [inline source/Various/File2.lagda:29]: Could you rename `𝕄-dont-contain-themselves` to `⁅-is-irreflexive` please?

File: source/Various/File1.lagda
Comment (tomdjong) [inline source/Various/File1.lagda:25]: We would prefer to use `𝟘-elim`, although we are that in some places in TypeTopology absurd patterns are still used.

File: source/Various/File2.lagda
Comment (tomdjong) [reply source/Various/File2.lagda:29]: We like this clever proof :slightly_smiling_face:

File: source/Various/File2.lagda
Comment (tomdjong) [inline source/Various/File2.lagda:69]: Could you indent lines 69 and 70 by one more space please?

**Review by tomdjong — CHANGES_REQUESTED** [review]
Thanks for your contribution @Seiryn21! In addition to the few minor requests, we (@martinescardo and I) ask you to merge the two files into a single file called `Finite.lagda` inside the `Iterative` folder, and to add `import Iterative.Finite -- by Alice Laroche` to `Iterative/index.lagda`.

After that, I can merge this quickly. Apologies for the delay!!

**Comment by martinescardo** [comment]
Let me have a quick look.

**Review by martinescardo — COMMENTED** [review]
Wonderful! Thanks a lot.

**Comment by martinescardo** [comment]
I am happy for you to merge, @tomdjong

**Pull request merged**
